diff.py

```diff
@@ -2538,14 +2565,20 @@ class ArchSettings:
     re_large_imm=re.compile(r"-?[1-9][0-9]{2,}|-?0x[0-9a-f]{3,}"),
     re_sprel=re.compile(r"-?(0x[0-9a-f]+|[0-9]+)(?=\((%ebp|%esi)\))"),
     re_imm=re.compile(r"-?(0x[0-9a-f]+|[0-9]+)|([\?$_][^ \t,]+)"),
-    re_reloc=re.compile(r"R_386_|dir32|DISP32|WRTSEG|OFF32|OFFPC32"),
+    re_reloc=re.compile(
+        r"R_386_|dir32|DISP32|WRTSEG|OFF32|OFFPC32|OFF16|OFFPC16|:\s+3\s"
```

Comment:
huh, what's up with the `:\s+3\s`?

Reply:
````  1a:   push   %ax
  1b:   push   %ds
  1c:   mov    $0x0,%ax
                        1d: OFF16       _tpal
                        1d: WRTSEG      DGROUP
  1f:   push   %ax
  20:   lcall  $0x0,$0x0
                        21: 3   _write_pels
  25:   add    $0x8,%sp```

Reply:
lcall somehow emit relocs with '3' instead of a named reloc type